mix.exs

```diff
@@ -33,9 +33,16 @@ defmodule Membrane.MP4.Plugin.MixProject do
       main: "readme",
       extras: ["README.md"],
       source_ref: "v#{@version}",
-      nest_modules_by_prefix: [Membrane.MP4],
+      nest_modules_by_prefix: [
+        Membrane.MP4,
+        Membrane.MP4.Box,
```

Comment:
Back to the modules again: I don't like how these boxes are now separated from the `Container` now.
1. I'd still insist on using `Box` as a suffix, e.g. `TrakBox`. This will not strip the context after aliasing
2. It seems that `Container`, `Track` and boxes should belong to the same namespace, so either put boxes under `Container` or make it flat as suggested originally